lib/book.rb

```diff
@@ -0,0 +1,31 @@
+class Book
+  attr_accessor :read
+  attr_reader :reviews
+  def initialize(book)
+    @book = book
```

Comment:
We should be splitting up book here  and I'd set up something like `@title = book[:title]`

Reply:
Mike, would you mind explaining this one a little more? My thought was that we wouldn't want to lose the original data and and instead store it as a whole, then access the data within as we need it.